binding/python/_orca.py

```diff
@@ -265,50 +420,70 @@ def sample_rate(self) -> int:
     def max_character_limit(self) -> int:
         """Maximum number of characters allowed in a single synthesis request."""
 
-        return self._max_character_limit
+        c_max_character_limit = c_int32()
+
+        status = self._max_character_limit_func(self._handle, byref(c_max_character_limit))
+        if status is not PicovoiceStatuses.SUCCESS:
+            raise _PICOVOICE_STATUS_TO_EXCEPTION[status](
+                message="Unable to get Orca maximum character limit",
+                message_stack=self._get_error_stack())
+
+        return c_max_character_limit.value
```

Comment:
if it's constant, maybe better to get its value at init and here just return that value